build-system/tasks/process-github-issues.js

```diff
@@ -108,6 +108,11 @@ function updateGitHubIssues() {
     getIssues(3),
```

Comment:
I think you can get as many issues as you want in one go by changing `per_page`. See https://developer.github.com/v3/guides/traversing-with-pagination/#changing-the-number-of-items-received

If that doesn't work, create a `const` called `NUM_PAGES` at the top of this file, and use it here instead of hard-coding each page fetch .

Reply:
it's number of pages actually, its batches. I need to request them selectively as specified and cannot make it a constant, but added a comment. let me know if it makes sense.

Reply:
What I meant was that you should be able specify how many issues are downloaded in a batch via `per_page`. So instead of fetching `X` issues `Y` times, you'd simply do a single fetch for `X * Y` items by changing the batch size. See the link above.

If that's not possible, or if the upper limit of the batch size doesn't meet your requirements, we should generate all the `getIssues` promises below from inside a single loop, by first constructing an array where you push `getissues(i)` a total of 10 times to the array, with `i` == 1 through `NUM_PAGES`, where `NUM_PAGES` is a constant equal to `10`. That's better than hard-coding ten consecutive promises.